packages/@glimmer/compiler/lib/javascript-compiler.ts

```diff
@@ -182,6 +236,10 @@ export default class JavaScriptCompiler<T extends TemplateMeta> {
     this.template.block.yields.add(to);
   }
 
+  debugger() {
+    this.push(['debugger', null, null]);
+  }
```

Comment:
No specific reason. I probably should have had a better understanding of the system before implementing. Have to look at the specialization stuff.